lms/templates/emails/enroll_email_allowedmessage.txt

```diff
@@ -18,16 +18,16 @@ ${_("To access the course visit {course_about_url} and register for the course."
 % endif
 % else:
 
-${_("To finish your registration, please visit {site_name}/dashboard\n"
-        "You will need to login with Touchstone to access this page. If you "
-        "do not have a Touchstone account, please contact your instructor for "
-        "detailed directions.").format(
+${_("To access the course, you will first need to register for MITx at {site_name}"
+        "with your Kerberos ID or Touchstone account."
+        "If you do not have a Kerberos ID and need to set up a Touchstone account,
+        "please contact your instructor for detailed directions.\n").format(
             site_name=site_name
     )}
 % if auto_enroll:
-${_("Once you have logged in with Touchstone, you will see "
-    "{course_name} listed on your dashboard.").format(
-        course_name=display_name or course.display_name_with_default_escaped
+${_("Once you have registered and logged in, visit the course site at "
+    "{course_url} to complete your enrollment.").format(
```

Comment:
Do we need more detail about how to "complete your enrollment" ?

Reply:
If they are sent to the {course_URL}/about, they'll land on the page that has the Click Here to Enroll button.

Reply:
The button on those pages says "Enroll in [Course #]"  

I thought that would be enough. We could add "complete your enrollment by clicking the Enroll button."

Reply:
I think adding the /about will do the trick.